zebra-chain/src/block/genesis/block-regtest-0-000-000.txt

```diff
@@ -0,0 +1 @@
+040000000000000000000000000000000000000000000000000000000000000000000000db4d7a85b768123f1dff1d4c4cece70083b2d27e117b4ac2e31d087988a5eac40000000000000000000000000000000000000000000000000000000000000000dae5494d0f0f0f2009000000000000000000000000000000000000000000000000000000000000002401936b7db1eb4ac39f151b8704642d0a8bda13ec547d54cd5e43ba142fc6d8877cab07b30101000000010000000000000000000000000000000000000000000000000000000000000000ffffffff4d04ffff071f0104455a6361736830623963346565663862376363343137656535303031653335303039383462366665613335363833613763616331343161303433633432303634383335643334ffffffff010000000000000000434104678afdb0fe5548271967f1a67130b7105cd6a828e03909a67962e0ea1f61deb649f6bc3f4cef38c4f35504e51ec112de5c384df7ba0b8d578a4c702b6bf11d5fac00000000
```

Comment:
It would be nice to have them all in one place, but it still seems wrong to put it in `zebra_test::vectors` since it's being used outside of Zebra's tests, @oxarbitrage could we leave this for now and potentially move them all to a more generically named folder like `block_data` later?

Reply:
> it still seems wrong to put it in `zebra_test::vectors` since it's being used outside of Zebra's tests

I agree. I didn't realize this.